app/server/lib/ActiveDoc.ts

```diff
@@ -403,7 +405,7 @@ export class ActiveDoc extends EventEmitter {
     this._attachmentFileManager = new AttachmentFileManager(
       this.docStorage,
       _attachmentStoreProvider,
-      _options?.doc,
+      forkId ? { id: forkId, trunkId, } : { id: trunkId, trunkId: undefined },
```

Comment:
There was an issue in tests where `options.doc` wasn't provided, meaning we couldn't use external attachments. 

This is a much more robust way of getting this information, as _docName (which trunkId and forkId come from) is a required parameter.